samples/CommunityToolkit.Maui.Sample/AppShell.xaml.cs

```diff
@@ -172,7 +172,7 @@ public static string GetPageRoute(Type viewModelType)
 		return uri.Uri.OriginalString[..^1];
 	}
 
-	static string GetPageRoute(Type galleryPageType, Type contentPageType) => $"/{galleryPageType.Name}/{contentPageType.Name}";
+	static string GetPageRoute(Type galleryPageType, Type contentPageType) => $"//{galleryPageType.Name}/{contentPageType.Name}";
```

Comment:
I'm good with this change but I would just like to confirm with @brminnick as he made the original change. Was there something you were fixing that this will reintroduce?

Reply:
I'm totally cool with it! Apologies - I didn't realize I had broken the Windows app when I did it. 

Long story short, I'm working on [porting my app, GitTrends, from Xamarin to MAUI](https://github.com/brminnick/GitTrends/pull/783), and I found a bug where routing wasn't working with the double slashes in the Shell path. So I "fixed" it here in the sample app too. BUT - it sounds like maybe there's a bug in my code (and perhaps my understanding of Shell).